tools/rna_tools/mqc/mQC.xml

```diff
@@ -0,0 +1,109 @@
+<tool id="mQC" name="mQC" version="1.2">
+    <description>quality control of ribosome profiling mapping results</description>
+    <requirements>
+        <requirement type="set_environment">SCRIPT_PATH</requirement>
+        <requirement type="package" version="1.3.1">samtools</requirement>
+        <requirement type="package" version="1.2">mqc</requirement>
+    </requirements>
+    <command interpreter="perl">
+        mQC.pl --galaxy Y --samfile $samfile --experiment_name $exp_name --cores $cores --species $species --ens_v $ensv --unique $uniq --mapper $mapper --maxmultimap $maxmultimap --ens_db "get" --offset $offsetSelect.offset --min_length_gd $min_l_gd --max_length_gd $max_l_gd --tool_dir "/srv/galaxy/server/tool-data/mQC/mqc_tools/" --plotrpftool $plotrpftool --outhtml $MappingQC_output --outfolder $MappingQC_output.files_path
+        
+        #if $offsetSelect.offset == "from_file"
+        
+        --offset_file $offsetSelect.offsetFile
+        
+        #else if $offsetSelect.offset == "plastid"
+        
+        --plastid_bam $offsetSelect.plastid_bam
+        --min_length_plastid $offsetSelect.min_l_plastid
+        --max_length_plastid $offsetSelect.max_l_plastid
+        
+        #end if
+        
+    </command>
+    <inputs>
+        <param name="samfile" type="data" format="sam" label="Choose the sam file on which you want to do quality control"/>
```

Comment:
I could include an option that you can input a BAM file and that mQC uses samtools to convert this input BAM file to a SAM file to work on. Is this desirable?
An what do I put then in the format="sam" option of the XML file?